src/main.rs

```diff
@@ -122,23 +124,25 @@ fn main() {
             desktop_window
         );
 
+        let xoff = app_config.x_offset;
```

Comment:
I don't like how you cal lit `xoff` here and `x_offset` otherwise. I know it makes the formulas a little bit longer but I think that's an ok tradeoff for general readability. Could you change it to `x_offset` and `y_offset`?

Reply:
I agree. Fixed.